### ISSUE TITLE
[4.0.0-RC1] The console help command shows an error for `contao:automator`

### ISSUE THREAD

**Issue by xchs** [contao/core-bundle#272]
The Symfony console help outputs an PHP error for the following Contao command:
```shell
$ app/console help contao:automator
```
```console
PHP Fatal error:  Method Contao\CoreBundle\Command\AutomatorCommand::__toString() must not throw an exception in /var/www/contao4/develop/vendor/symfony/symfony/src/Symfony/Component/Console/Descriptor/TextDescriptor.php on line 0
[2015-05-16 02:25:31] php.EMERGENCY: Fatal Error: Method Contao\CoreBundle\Command\AutomatorCommand::__toString() must not throw an exception {"type":1,"file":"/var/www/contao4/develop/vendor/symfony/symfony/src/Symfony/Component/Console/Descriptor/TextDescriptor.php","line":0,"level":-1,"stack":[]}

  [Symfony\Component\Debug\Exception\FatalErrorException]
  Error: Method Contao\CoreBundle\Command\AutomatorCommand::__toString() must not throw an exception
```

**Comment by aschempp** [comment]
@leofeyer I guess the framework service was not injected. Voting again for constructor injection ;-)

**Comment by leofeyer** [comment]
It does not matter how the service is injected. If we don't initialize it in the `generateCommandMap()` method, it will never work :)

Fixed in e618cf4103b8316a1c31d6c9a831783d83a376fe.
